### PR TITLE
Clarify source override usage

### PR DESCRIPTION
**Pull request by e271828-**

### CODE REVIEW

File: sdk/src/main/java/com/hcaptcha/sdk/HCaptchaConfig.java [hunk 1]
```diff
@@ -41,32 +41,32 @@
 
     /**
      * The url of api.js
-     * Default: https://hcaptcha.com/1/api.js
+     * Default: https://js.hcaptcha.com/1/api.js (Override only if using first-party hosting feature.)
      */
     @Builder.Default
     private String apiEndpoint = "https://hcaptcha.com/1/api.js";
```
Comment (DSergiu) [inline sdk/src/main/java/com/hcaptcha/sdk/HCaptchaConfig.java:47]: Should the value be updated as well?

**Review by DSergiu — APPROVED** [review]

File: sdk/src/main/java/com/hcaptcha/sdk/HCaptchaConfig.java (hunk 1 above)
Comment (e271828-) [inline sdk/src/main/java/com/hcaptcha/sdk/HCaptchaConfig.java:47]:
```suggestion
    private String apiEndpoint = "https://js.hcaptcha.com/1/api.js";
```

Comment (e271828-) [reply sdk/src/main/java/com/hcaptcha/sdk/HCaptchaConfig.java]: Yep :)

**Review by cryptoroland — COMMENTED** [review]
[](https://github.com/hCaptcha/hcaptcha-android-sdk/commit/ab0c233c8d6d24230356d9ee2f6ded42394634ba)

**Review by cryptoroland — COMMENTED** [review]
[](https://github.com/hCaptcha/hcaptcha-android-sdk/pull/9/files)

**Pull request merged**